src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py

```diff
@@ -337,7 +337,7 @@ def loss_layer(self, predict, labels):
         kr_obj_ij = _tf.stop_gradient(resp_box * gt_conf)
 
         kr_noobj_ij = 1 - kr_obj_ij
-        s = 1 / (self.batch_size * self.grid_shape[0] * self.grid_shape[1])
+        s = 1 / (batch_size * self.grid_shape[0] * self.grid_shape[1])
```

Comment:
What is `s`?

Reply:
It is a normalization constant for the loss function I guess ;)